### PR TITLE
test:Remove unused HttpHelpers

### PR DESCRIPTION
**Pull request by marcotc**
It was introduced here https://github.com/DataDog/dd-trace-rb/pull/369/files#diff-ffc89a0ffd0de70be571ade8e3296d6a96c3126da82421877a6027b533d72773R149, but it's not longer used.

I don't see much future use for a stand-alone mock request like this, since we use WebMock and rack-test today when this need arises.

### CODE REVIEW

**Comment by ivoanjo** [comment]
You forgot to remove a `require` for that file ;)

**Review by ericmustin — CHANGES_REQUESTED** [review]
small 1liner otherwise lgtm

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -48,7 +48,6 @@
   config.include ConfigurationHelpers
   config.include ContainerHelpers
   config.include HealthMetricHelpers
-  config.include HttpHelpers
```
[the remark this answers is not in the document]
Comment (ericmustin) [reply spec/spec_helper.rb:51]: ugh link isn't working, but `require 'support/http_helpers'` shud get removed

**Review by ericmustin — APPROVED** [review]

**Comment by marcotc** [comment]
I was wrong, this is used 🦆

**Pull request closed without merging**